decidim-meetings/app/commands/decidim/meetings/admin/create_meeting.rb

```diff
@@ -23,6 +23,8 @@ def call
 
         def create_meeting
           Meeting.create!(
+            decidim_scope_id: @form.decidim_scope_id,
+            decidim_category_id: @form.decidim_category_id,
```

Comment:
I don't like using ids here, see `Admin::ProposalForm` since it also has scope & category.